### ISSUE TITLE
Add `aria-required` to Support admin - manage users page's "Organization access" tab

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#8543]
## Area or Component

`/app/admin/manage-users` under the "Organization access" tab after searching for a user

## Quick Summary

"Organization access" combo box is required but does not have that attribute

## Accessibility Violation

[Required field missing aria-required
](https://docs.deque.com/issue-help/1.0.0/en/aria-required-missing)
## Action Requested

- Either [use our combobox to pass in the `required` value](https://github.com/CDCgov/prime-simplereport/blob/4f2d9ee0ed0b279e23ab5086a0cea1f616719d18/frontend/src/app/commonComponents/ComboBox.tsx#L36) or do [something like this](https://github.com/CDCgov/prime-simplereport/blob/4f2d9ee0ed0b279e23ab5086a0cea1f616719d18/frontend/src/app/supportAdmin/DeviceType/DeviceForm.tsx#L227)

## Screenshots (optional)
<img width="1340" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM2tKQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--36f95840202eb3e4c5e793d00252beb9cea50fc6/Screenshot%202025-02-26%20at%2016.27.39.png" alt="Screenshot 2025-02-26 at 16.27.39.png" />

## Source Code (optional)

https://github.com/CDCgov/prime-simplereport/blob/4f2d9ee0ed0b279e23ab5086a0cea1f616719d18/frontend/src/app/commonComponents/UserDetails/UserOrganizationFormField.tsx#L83
